### ISSUE TITLE
GB Spring bank holiday marked as Platinum Jubilee (also a potential issue with Easter Sunday)

### ISSUE THREAD

**Issue by VirtualDOMinic** [commenthol/date-holidays#303]
https://github.com/commenthol/date-holidays/blob/cff4a080db1421728dddc777bb4ae6e852ea401b/data/countries/GB.yaml#L59-L61

Hi,

Thanks for this useful library!

There is a mistake due to some confusion with the setting of the Platinum Jubilee as [the article you've referenced](https://www.gov.uk/government/news/extra-bank-holiday-to-mark-the-queens-platinum-jubilee-in-2022) is a little hard to parse.

The platinum jubilee bank holiday is **only** Friday 3rd June, as per https://www.gov.uk/bank-holidays (a clearer place to view this). Thursday 2nd June should instead be marked as the "Spring Bank Holiday". I might manage to make my own PR for this, but am unsure as to when (other things at work taking precedent) so a PR from somebody else is very welcome.

As an aside, I'm unsure as to whether it is appropriate to have Easter Sunday marked as a public holiday (perhaps it should just be "observance"?) as per [this article](https://www.timeanddate.com/holidays/uk/easter-sunday) and [this one](https://www.publicholidayguide.com/bank-holiday/easter-sunday-bank-holiday/), and its notable absence from the gov uk bank-holidays link above.

**Comment by commenthol** [comment]
Hi @VirtualDOMinic, thanks for rising this issue. Can you please review the changes in #304?

**Comment by VirtualDOMinic** [comment]
Thanks @commenthol ! Looking over it now. Appears that CI is failing due to outdated snapshots, FYI

**Comment by commenthol** [comment]
Thanks for the reminder. Fixtures are up-to-date now and CI is fine again.
